specs/IsPointInDraggableRegion.md

```diff
@@ -0,0 +1,303 @@
+
+-->
+
+Draggable Regions For Visual Hosted Apps
+===
+
+# Background
+<!-- TEMPLATE
+    Use this section to provide background context for the new API(s)
+    in this spec. Try to briefly provide enough information to be able to read
+    the rest of the document.
+
+    This section and the appendix are the only sections that likely
+    do not get copied into any official documentation, they're just an aid
+    to reading this spec. If you find useful information in the background
+    or appendix consider moving it to documentation.
+    
+    If you're modifying an existing API, included a link here to the
+    existing page(s) or spec documentation.
+
+    For example, this section is a place to explain why you're adding this
+    API rather than modifying an existing API.
+
+    For example, this is a place to provide a brief explanation of some dependent
+    area, just explanation enough to understand this new API, rather than telling
+    the reader "go read 100 pages of background information posted at ...". 
+-->
+Draggable regions are marked regions on a webpage that will move the app window when the user clicks and drags on them. It also opens up the system menu if right-clicked. This API is designed to provide draggable regions support for Visual Hosting.
+
+
+# Examples
+## Enable Dragging (L button)
+### Handle WM_NCHITTEST
+The objective is to tell Windows that the current mouse coordinate corresponds to a caption location if it is within a draggable region.
+#### Win32
+```cpp
+       case WM_NCHITTEST:
+            POINT pt {GET_X_LPARAM(lParam), GET_Y_LPARAM(lParam)};
+            BOOL isInRegion = FALSE;
+            webview2->IsPointInDraggableRegion(&pt, &isInRegion);
+
+            if(isInRegion)
+            return HTCAPTION;
+    
+            break;
+```
+#### .NET / WinRT
+```c#
+        if (m.Msg == WM_NCHITTEST)
+        {
+            Point p = new Point(GET_X_LPARAM(m.lParam), GET_Y_LPARAM(m.lParam));
+
+            if (webview2.IsPointInDraggableRegion(ref p)) {
+                m.Result = (IntPtr)HTCAPTION;
+            }
+        }
+```
+## Enable System Menu (R button)
+### Handle WM_NCRBUTTONDOWN
+NOTE: The setup above is required as well.
+
+Here we must capture the mouse if the current message corresponds to an HTCAPTION area. For later use, we must also flag that we have received an R button down on a draggable region. The latter part is done using 
+
+```cpp
+ncrbuttondown_on_draggable_region_ = TRUE
+```
+
+NOTE: It is not necessary, but recommended that you also check to ensure the coordinates here lie in a draggable region. Not doing so opens up the possibility that the default system menu will always be overridden. This may or may not be an issue for your application. 
+
+#### Win32
+```cpp
+        case WM_NCRBUTTONDOWN:
+            BOOL isInRegion = FALSE;
+            webview2->IsPointInDraggableRegion(&pt, &isInRegion);
+            if(
```

Comment:
The coding style should be more like:

// space between 'if' and '('
if (something == 1 // conditions on same line as if
    && otherThing != 2) // If you need to wrap the condition, then indent
{ // braces on their own lines matching indent of the 'if'
    ... // code inside the block is indented 4 spaces
}

Please apply to sample code.